core/primitives/src/views.rs

```diff
@@ -32,31 +34,56 @@ use crate::types::{
 /// A view of the account
 #[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize, Debug, Eq, PartialEq, Clone)]
 pub struct AccountView {
+    /// A spendable balance (before state rent charge)
     #[serde(with = "u128_dec_format")]
     pub amount: Balance,
+    /// An actual balance in storage
```

Comment:
That's wrong comment. It sounds like it's the value from the storage, while it's computed value. Maybe you should mention that this is a value if we charge rent at the current block height.

Reply:
changed the structure see
https://github.com/nearprotocol/nearcore/pull/1974#discussion_r370602718